client/my-sites/hosting/sftp-card/test/index.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * @jest-environment jsdom
+ */
+
+/**
+ * External dependencies
+ */
+import { shallow, mount } from 'enzyme';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { SftpCard } from '..';
+
+const translate = x => x;
+const requestSftpUsers = x => x;
+
+describe( 'SftpCard', () => {
+	describe( 'Sftp Questions', () => {
+		it( 'should display sftp questions if no sftp username', () => {
+			const wrapper = shallow( <SftpCard translate={ translate } /> );
+
+			expect( wrapper.find( '.sftp-card__questions' ).length ).toEqual( 1 );
```

Comment:
Could we maybe use `toHaveLength` here?
```suggestion
			expect( wrapper.find( '.sftp-card__questions' ) ).toHaveLength( 1 );
```

Reply:
yep, thanks - I couldn't see that method for looking in the enzyme api yesterday!